sections/dashboard/Stake/TradingRewardsTab.tsx

```diff
@@ -101,25 +109,23 @@ const TradingRewardsTab: React.FC<TradingRewardProps> = ({
 			? claimableRewards.reduce((acc, curr) => acc + Number(curr[2]) / 1e18, 0)
 			: 0;
 
-	const SpotFeeQuery = useGetSpotFeeForAccount(walletAddress!, start, end);
-	const spotFeePaid = useMemo(() => {
-		const t = SpotFeeQuery.data?.synthExchanges ?? [];
+	const futuresFeeQuery = useGetFuturesFeeForAccount(walletAddress!, start, end);
+	const futuresFeePaid = useMemo(() => {
+		const t = futuresFeeQuery.data ?? [];
 
 		return t
-			.map((trade: any) => Number(trade.feesInUSD))
+			.map((trade: any) => Number(trade.feesPaid) / 1e18)
```

Comment:
could we use the correct type here instead of `any` and probably better to keep values in wei or BigNumber for arithmetic of these number sizes / decimals.

Reply:
Yes. I will go through all the similar cases and use the correct type. Thanks for pointing it out.